gatsby-node.js

```diff
@@ -75,136 +76,137 @@ const createCareerPages = async ({ actions }) => {
     };
     const jobsParse = xmlParser.parse(jobsXml, options);
     const positions = jobsParse['workzag-jobs'].position;
-    return { positions, langKey };
+    return { positions, languageKey };
   });
 
-  await Promise.all(positionsGroupedByLanguage).then(async (values) => {
-    for (let i = 0; i < values.length; i++) {
-      const positions = values[i].positions;
-      const langKey = values[i].langKey;
+  await Promise.all(fetchPositionsByLanguage).then(
+    async (positionsByLanguage) => {
+      for (let index = 0; index < positionsByLanguage.length; index++) {
+        const positions = positionsByLanguage[index].positions;
+        const languageKey = positionsByLanguage[index].languageKey;
 
-      // position is valid if it has a job description and slug
-      const isValidPosition = (position) => {
-        return (
-          position.jobDescriptions.jobDescription &&
-          Array.isArray(position.jobDescriptions.jobDescription)
-        );
-      };
+        // position is valid if it has a job description and slug
+        const isValidPosition = (position) => {
+          const { jobDescription } = position.jobDescriptions;
+          return Array.isArray(jobDescription);
+        };
 
-      const getSlug = (position) => {
-        if (position.satellytesPath) return position.satellytesPath;
-        const slug = position.jobDescriptions.jobDescription.find(
-          (description) =>
-            description.name.trim() === 'Slug' && description.value.trim(),
-        );
-        return `/career/${slug.value.trim()}/`;
-      };
+        const getSlug = (position) => {
+          if (position.satellytesPath) return position.satellytesPath;
```

Comment:
Single line `if` are hard to read. They also should follow with an empty line so that they can be seen at the first sight.

```
if (position.satellytesPath) return position.satellytesPath;
const slug = position.jobDescriptions.jobDescription.find()
```

vs.

```

if (position.satellytesPath) {
  return position.satellytesPath;
}

const slug = position.jobDescriptions.jobDescription.find()
```